building/build-debs/homeworld-admin-tools/build-package.sh

```diff
@@ -11,5 +11,10 @@ git rev-parse HEAD | tr -d '\n' >resources/GIT_VERSION
 [[ -n "$(git status --porcelain)" ]] && echo -n "-dirty" >>resources/GIT_VERSION
 echo >>resources/GIT_VERSION
 
+echo "${HOMEWORLD_APT_BRANCH}" >>resources/APT_BRANCH
+
+HOMEWORLD_APT_SIGNING_KEY="$(cd ../.. && source get-apt-signing-key.sh)"
```

Comment:
Why does this need to be sourced? Why do we need to change directories to use it?

Any reason not to make it a function instead of an entire script?